app/assets/v2/css/toolbox.css

```diff
@@ -25,7 +25,46 @@
   font-size: 1.4em;
 }
 
+#Faucet img {
+  margin-top: 40px;
+}
+
+
+#toc{
+  position: fixed;
+  top: 150px;
+  left: 50px;
+  border-top: 1px black solid;
+  border-left: 2px black solid;
+  background-color: #f5f5f5;
```

Comment:
Personal preference is to use all uppercase: `#F5F5F5` I find it helps with legibility

Reply:
word we should perhaps add this to the style guide